airbyte-ci/connectors/metadata_service/lib/metadata_service/gcs_upload.py

```diff
@@ -262,3 +262,91 @@ def upload_metadata_to_gcs(
             ),
         ]
     )
+
+def upload_all_docs_to_gcs(connectors_dir: Path, docs_dir: Path, bucket_name: str):
```

Comment:
📚 Lets write a docstring explaining what it does, why we added it, and if theres any criteria to remove this